assets/js/tabs.js

```diff
@@ -15,19 +15,24 @@
   var interval = 3000;
   var tabCarousel = setInterval(nextSlide, interval);
 
-  $('.tab-content.tab-focus, .nav-tabs.nav-tabs-focus').hover(function(e){
+  $(document).on({
```

Comment:
On `document`, really ?
This is weird. So if I understood correctly the behavior you just implemented is:
> When my cursor is on the whole page, the carrousel stop. And when I move my cursor "out of the page (out of the window)", the carrousel start again.